examples/incompressible-flow/2d-flow-around-cylinder/cylinder-structured.geo

```diff
@@ -99,9 +99,9 @@ Line(52)={22,32};
 Line(53)={21,31};
 
 //Theta of the circle
-Transfinite Line {3,22,4,10,34,20,32,13,40,41,42,43} = Ceil(nt) Using Progression 1.0;
+Transfinite Line {3,22,4,10,34,20,32,13,40,41,42,43,63,33,35,66,60,30,31,64,2,12,11,1} = Ceil(nt) Using Progression 1.0;
 // Radial direction
-Transfinite Line {50,51,52,53} = Ceil(nr) Using Progression 1.05;
+Transfinite Line {50,51,52,53} = Ceil(nr) Using Progression 1.0;
 Transfinite Line {5,21,23,6} = Ceil(nl) Using Progression 1.00;

```

Comment:
Why is Ceil here? I think nr and nl already are integers

Reply:
Good question I have no clue. The geo file was already there.